components/MyTeamMembers.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="my-team__members">
+    <article v-for="member of members" :key="member" class="member-card">
+      <img class="member-card__photo" :src="member.photoUrl" alt="" />
+      <div class="member-card__job-title">
+        {{ member.jobTitle }}
+      </div>
+      <h2 class="member-card__name">
+        {{ member.name }}
+      </h2>
+    </article>
+  </div>
+</template>
+<script>
+export default {
+  name: 'MyTeamMembers',
+  data() {
+    return {
+      members: [],
+    };
+  },
+  async fetch() {
```

Comment:
Why do you need to do a fetch here for `/members` if you already did it in the parent component? Maybe you can remove this one get the data from the parent element